pkg/controller/dnsendpoint/nameserver/gcp_live_test.go

```diff
@@ -139,3 +145,26 @@ func (s *LiveGCPTestSuite) getCUT() *gcpQuery {
 		},
 	}
 }
+
+func (s *LiveGCPTestSuite) testCreateThenUpdate(tc *testCreateAndDeleteCase) {
```

Comment:
Use a different type for the parameter to this function. Something like `testCreateThenUpdateCase`.